atlp-rwanda/rca-phantom-team1-bn
# Feat/enquire bus info

## What does this PR do
- It implements the enquire bus info
- It fixes the issues raised on CRUD for buses

## Description
- The user should be able to retrieve details of a bus so that they can plan their next move accordingly.
- Buses can be created, updated, and searched using plate numbers.
- A bus that is fetched from the database allows the user to view the total number of commuters in the bus, bus type, and available seats.

## How to manually test this 
- Check in the branch `feat/enquire-bus-info`
- Run the server
- Open the Swagger API documentation to test the buses as a user

## Relevant Trello story 
[[11-crud-operations-for-buses][(https://trello.com/c/a6wcZ3PE/9-create-roles-and-set-permissions)](https://trello.com/c/maUQgDhP/11-crud-operations-for-buses)](https://trello.com/c/wG92lKKE/16-enquire-buses-info)

## Screenshots
![image](https://user-images.githubusercontent.com/53254885/230772742-b6502e43-10be-412a-bf1f-83a83a535196.png)

![image](https://user-images.githubusercontent.com/53254885/230772751-fc700fb8-182e-461a-95fc-8fa48822d98f.png)
